### DISCUSSION TITLE
How to convert square pixels into cm^2

### DISCUSSION THREAD

**Discussion by nlopezfigueroa** [biigle/core#559] [Q&A]
Hello!

I am currently using BIIGLE to calculate percentage coverage of benthic jellyfish in seagrass meadows. I am having a hard time to do this because I am not sure how to obtain measurements on a unit other than pixels. Any help with this topic will be much appreciate it to finish my dissertation.

**Comment by mzur** [comment]
*Disclaimer: I'm not a biologist who does this kind of thing for a living. This is just my technical point of view.*

BIIGLE requires image area information to estimate the area of annotations. Image area information can either be obtained through [laser point detection](https://biigle.de/manual/tutorials/laserpoints/laserpoint-detection) or supplied with [image metadata](https://biigle.de/manual/tutorials/volumes/file-metadata). Please be aware that in all cases the area estimation assumes that the annotations were done on a flat surface (e.g. the sea floor) and the camera points straight down to the surface. If the camera angle is different and/or the surface is not flat, the area estimation will be inaccurate.

If you use a photo quatrat, you can do it in a different way. First, create an annotation for the whole quadrat (e.g. with a polygon). This will give you the information that e.g. 4000 px² correspond to 1 m² in the image. Then you can draw the individual annotations for coverage estimation. If you have an annotation with 400 px², for example, this would correspond to 0.1 m². However, this *still* assumes that the camera points straight down to the quadrat and the image is not skewed e.g. by the camera optics (i.e. the quadrat borders are not curved).

So you see that a pixel-accurate area estimation may be quite hard to obtain this way. More preferrable with photo quatdrats could be a coarser area estimation on the level of the individual quadrat sections (i.e. the "net" of the quadrat). You know how large each of these sections is in m². You can then just place a point annotation in each of the sections to say "this section contains mostly substrate x". Then annotate enough images and you should get a robust estimate.
